packages/react-components/src/components/__tests__/table-renderer-components.test.tsx

```diff
@@ -18,7 +18,7 @@ afterEach(() => {
 
 describe('Table output component tests', () => {
     
-    test('Renders AG-Grid table with provided props & state', async () => {
+    test.skip('Renders AG-Grid table with provided props & state', async () => {
```

Comment:
Add a comment about why skipping, similar to the other ones added in?

Reply:
Indeed. Fixed.